src/codeartifact/model.ts

```diff
@@ -0,0 +1,181 @@
+/*!
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as nls from 'vscode-nls'
+const localize = nls.loadMessageBundle()
+
+import * as vscode from 'vscode'
+import { CodeArtifact as CA } from 'aws-sdk'
```

Comment:
nit: avoid aliasing imports when possible
```suggestion
import { CodeArtifact } from 'aws-sdk'
```

Reply:
This was done to avoid the clash with the  CodeArtifact class.